Src/ElasticScale.Client/ShardManagement/ShardMap/ShardMapUtils.cs

```diff
@@ -95,24 +97,13 @@ static ShardMapUtils()
 
         /// <summary>
         /// Validate .Net runtime 4.6 if Active Directory Aughentication is specified in connection string.
+        /// Below call will throw if connection string specifies Active Directory authentication and library
+        /// is not running with .NET 4.6.
```

Comment:
Sorry, I'm still not sure why this is necessary. In methods where we call ValidateAuthenticationInConnectionString, don't we create the SqlConnectionStringBuilder later anyway? So this is making a redundant extra SqlConnectionStringBuilder?